### PR TITLE
Add `google.cloud.container` API.

### PR DESCRIPTION
**Pull request by lukesneeringer**
@jonparrott I am working on a playbook for going from the output of the code generator to a new, staged API on google-cloud-python (with the intent to go back and automate as much as I can).

Things I want to call attention to:

  * I had to make edits in docstrings for `bigtable` and `api_core` to resolve new ambiguities because GKE introduces objects called `Cluster` and `Operation`.
  * I did a new thing where I symlinked the docs in `container/docs/` over to `docs/container`; the code generator does generate these documentation pages and it does not seem to make much sense to move them outside into the monorepo docs structure unless necessary.
  * I edited the monorepo's `README.rst`, `docs/index.rst`, `docs/requirements.txt`. Did I miss anything?

### CODE REVIEW

**Comment by jonparrott** [comment]
> I had to make edits in docstrings for bigtable and api_core to resolve new ambiguities because GKE introduces objects called Cluster and Operation.

Oh my god I can't wait to break this up into separate libraries so we don't have to deal with that.

File: container/setup.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: Implementation :: CPython',
+    ],
+    description='GAPIC library for the Google Container Engine API',
+    include_package_data=True,
+    long_description=long_description,
+    install_requires=install_requires,
+    license='Apache 2.0',
+    packages=find_packages(),
```
Comment (jonparrott) [inline container/setup.py]: This needs to explicitly exclude `tests*`.

Comment (lukesneeringer) [reply container/setup.py]: Thanks; will document fix.

**Comment by lukesneeringer** [comment]
> Oh my god I can't wait to break this up into separate libraries so we don't have to deal with that.

That is exactly what I was thinking.

File: container/docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,310 @@
+# -*- coding: utf-8 -*-
```
Comment (jonparrott) [inline container/docs/conf.py]: (No action required)

At some point we'll need to give this file more scrutiny (during the repo split) but for now it's fine.

File: container/MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1,4 @@
+include README.rst LICENSE
+recursive-include google *.json *.proto
+global-exclude *.py[co]
```
Comment (jonparrott) [inline container/MANIFEST.in]: You'll want to add `recursive-include tests *` above line 3.

File: container/setup.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
+"""A setup module for the GAPIC Google Container Engine API library.
```
Comment (jonparrott) [inline container/setup.py]: This needs a license header.

File: container/setup.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: Implementation :: CPython',
```
Comment (jonparrott) [inline container/setup.py]: I'd drop this specifier, as it's possible that alternatively implementations could be used.

File: container/setup.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    ],
+    description='GAPIC library for the Google Container Engine API',
```
Comment (jonparrott) [inline container/setup.py]: please add `zip_safe=False`.

File: container/setup.py [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    include_package_data=True,
+    long_description=long_description,
+    install_requires=install_requires,
+    license='Apache 2.0',
+    packages=find_packages(exclude=('tests*',)),
+    namespace_packages=['google', 'google.cloud'],
+    url='https://github.com/googleapis/googleapis')
```
Comment (jonparrott) [inline container/setup.py]: This url needs to point to this repo, or even better, to the docs.

File: container/setup.py (hunk 4 above)
Comment (lukesneeringer) [reply container/setup.py]: Not obviously shown for whatever reason, but I added one.

**Comment by lukesneeringer** [comment]
Ignoring the spanner failure in CI; a previous run passed.

**Pull request merged**